python/py-rangehttpserver/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        danvk RangeHTTPServer 1.2.0
+name                py-${github.project}
+
+github.tarball_from tags
+
+platforms           darwin
+license             MIT
+
+python.versions     35 36 37
+
+maintainers         {gmail.com:rubendibattista @rubendibattista} openmaintainer
+description         SimpleHTTPServer with support for Range requests
+
+long_description    $description
+checksums           rmd160  5f42f69b3b9356fbda3453acef53282bdc7318e7 \
+                    sha256  639e59134dff9dd39ee88750649a90ea9c76a9b1332b8334632fec2c8ca99e1f \
+                    size    8167
+
+
+if {${subport} ne ${name}} {
+    depends_lib     port:py${python.version}-requests
```

Comment:
this needs ```py-setuptools``` as build dependency since it does ```from setuptools import setup``` in the ```setup.py```

Reply:
you added ```py-setuptools``` under ```depends_lib```, that's not correct. It is not needed at runtime, only during install; so as I said it should be added as ```depends_build-append```